tr_TR.json

```diff
@@ -728,8 +728,8 @@
   "KEEP_VEHICLE_FIXED": "Aracı Tamir Edin",
   "KEEP_VEHICLE_FIXED_DESC": "Aracınızı aşınma ve yıpranmadan korur",
   "KEEP_VEHICLE_REPAIRED": "Aracı Onarılmış Tut açıksa, aracın sağlığını değiştirmek için devre dışı bırakın.",
-  "KICK": "Tekme atmak",
-  "KICKED": "tekmelendi",
+  "KICK": "Tekmele",
```

Comment:
I think "Oyundan At" or "Sunucudan At" would be a better term for kicking someone.